pkgs/servers/mail/listmonk/frontend.nix

```diff
@@ -1,44 +1,42 @@
-{ yarn2nix-moretea
+{ mkYarnPackage
 , fetchFromGitHub
 , fetchYarnDeps
 , meta
 }:
 
-yarn2nix-moretea.mkYarnPackage rec {
+mkYarnPackage rec {
   pname = "listmonk-frontend";
-  version = "2.5.1";
+  version = "3.0.0";
 
-  src = fetchFromGitHub {
+  src = "${fetchFromGitHub {
     owner = "knadh";
     repo = "listmonk";
     rev = "v${version}";
-    sha256 = "sha256-gCnIblc83CmG1auvYYxqW/xBl6Oy1KHGkqSY/3yIm3I=";
-  };
+    sha256 = "sha256-Bd8Wl0sXGor0wtibbuOZVv3r5tPYELRnZnx30GpuaUA=";
+  }}/frontend";
```

Comment:
Please use `sourceRoot`.

Reply:
it does not correctly set the cwd. I've cleaned it by using the src of the default.nix